ts/utils/configs.ts

```diff
@@ -7,11 +7,11 @@ export const configs = {
     AMOUNT_DISPLAY_PRECSION: 5,
     BACKEND_BASE_PROD_URL: 'https://website.api.0x.org',
     BACKEND_BASE_STAGING_URL: 'https://staging-website-api.0x.org',
-    BACKEND_BASE_DEV_URL: 'https://localhost:3001',
+    BACKEND_BASE_DEV_URL: 'https://website.api.0x.org',
     API_BASE_PROD_URL: 'https://api.0x.org',
     API_BASE_STAGING_URL: 'https://staging.api.0x.org',
     API_BASE_KOVAN_URL: 'https://kovan.api.0x.org',
-    API_BASE_DEV_URL: 'http://localhost:3000',
+    API_BASE_DEV_URL: 'https://api.0x.org',
```

Comment:
These should not be here, I will clean up and merge so `development` is not dirty

Reply:
thanks good catch! was testing late last night haha